### DISCUSSION TITLE
番茄小说网乱码

### DISCUSSION THREAD

**Discussion by jiaminxun** [F9y4ng/GreasyFork-Scripts#87] [问答专区(Question / Answer)]
https://fanqienovel.com/
番茄小说网首页，书库，排行榜众多页面乱码。
![QQ截图20220428151244](https://user-images.githubusercontent.com/57553009/165697664-3aeea463-228d-400e-83a1-e2366f062478.jpg)
![QQ截图20220428151300](https://user-images.githubusercontent.com/57553009/165697674-f5775fef-6bc1-443b-a565-5ea8e5932336.jpg)
![QQ截图20220428151313](https://user-images.githubusercontent.com/57553009/165697676-ca8b2bc9-8af5-43b4-a0ef-8e7daaf84c2c.jpg)

**Comment by F9y4ng** [comment]
* **方案一：** 如要使用自定义字体，则在 **需要渲染的网页元素** 中，追加 `:not([class*='font-'])`，保存为站点独享。
*  **方案二：** 如不需要自定义字体替换，关闭字体重写开关即可，保存为站点独享。

**Comment by jiaminxun** [comment]
使用第一种方案，首页正常了，书库页面和排行榜页面是没有乱码了，可是字体不是自定义的字体啊

**Reply by F9y4ng** [reply]
书库页面和排行榜页面显示乱码的地方就是网站默认定义的专用字体，方案一是将应用专用字体的样式过滤掉不进行字体渲染，那肯定就不是你要设置的自定义字体了。
